client/src/www/app/outline_server_repository/vpn.fake.ts

```diff
@@ -26,23 +26,23 @@ export class FakeVpnApi implements VpnApi {
 
   constructor() {}
 
-  private playBroken(hostname?: string) {
-    return hostname === FAKE_BROKEN_HOSTNAME;
+  private playBroken(address?: string) {
+    return address.startsWith(FAKE_BROKEN_HOSTNAME);
```

Comment:
`address` might be `undefined`.
```suggestion
    return address?.startsWith(FAKE_BROKEN_HOSTNAME);
```